sdk/core/ci.yml

```diff
@@ -31,7 +31,7 @@ stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
     parameters:
       ServiceDirectory: core
-      CtestRegex: azure-core
+      CtestRegex: AzureCore.
```

Comment:
This should match the "module" name (or what we would consider the "package" name). Just like we have `azure-identity`, can this be `azure-core`?

Reply:
updated to azure-core.  Makes sense.